projects/components/src/table/table.component.ts

```diff
@@ -512,8 +515,15 @@ export class TableComponent
     private readonly domElementMeasurerService: DomElementMeasurerService,
     private readonly tableService: TableService,
     private readonly modalService: ModalService,
+    private readonly fileDownloadService: FileDownloadService,
+    private readonly tableCsvDownloaderService: TableCsvDownloaderService,
+    private readonly notificationService: NotificationService,
     private readonly preferenceService: PreferenceService
   ) {
+    this.tableCsvDownloaderService.csvDownloadRequest$.pipe(filter(tableId => tableId === this.id)).subscribe(() => {
```

Comment:
subscribing to a long lived observable here, need to clean it up.

Reply:
fixed